setup.cfg

```
@@ -28,7 +28,7 @@ package_dir =
     = src
 packages = find_namespace:
 install_requires =
-    stactools ~= 0.1
+    stactools ~= 0.2.1a1
```

```suggestion
    stactools == 0.2.1a1
```

While we're in prerelease mode for stactools, a new prerelease could be a breaking change, so we should hard pin.